templates/aem-multimodule-project/lazybones.groovy

```diff
@@ -335,6 +341,100 @@ def installDir = new File(projectDir, "content/src/main/content/jcr_root/apps/${
 installDir.mkdirs()
 writeToFile(installDir, ".vltignore", "*.jar")
 
+// Creating AEM 6.2 Editable Templates folders 
+// if (props.aemVersion == VERSION_62 && props.createEditableTemplatesStructure) {
+if (props.createEditableTemplatesStructure) {
+    println "Creating AEM 6.2 Editable Templates folders..."
+
+	def confDir = new File(projectDir, "content/src/main/content/jcr_root/conf")
+	confDir.mkdirs()
+    writeToFile(confDir, ".content.xml", """\
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:rep="internal"
+    jcr:mixinTypes="[rep:AccessControllable]"
+    jcr:primaryType="sling:Folder"/>
+""")
+
+	def confProjectDir = new File(confDir, "${props.appsFolderName}")
```

Comment:
we can do it, but I think the value will be the same with ${props.appsFolderName}"